common/persistence/sql/sqlplugin/postgres/configstore.go

```diff
@@ -24,15 +24,37 @@ package postgres
 
 import (
 	"context"
-	"fmt"
 
+	"github.com/uber/cadence/common"
 	"github.com/uber/cadence/common/persistence"
+	"github.com/uber/cadence/common/persistence/sql/sqlplugin"
+)
+
+const (
+	_selectLatestConfigQuery = "SELECT row_type, version, timestamp, data, data_encoding FROM cluster_config WHERE row_type = $1 ORDER BY version LIMIT 1;"
```

Comment:
what do you think about adding `ORDER BY DESC`

Reply:
I prefer to keep the behavior the same as MySQL.